src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,311 @@ paths:
+          schema:
+            $ref: '#/definitions/subscribers_for_alert_by_assetId'
+        401:
+          description: "Unauthorized"
+        404:
+          description: "There is no alert associated with the provided assetId."
+          schema:
+            $ref: '#/definitions/get_alert_subscription_for_asset_id_error_message'
+        500:
+          description: "Internal Server Error"
+  '/alert-subscriptions/{assetId}/{alertType}':
+    get:
+      operationId: "listSubscribersForAnAlertByAssetIdAndAlertType"
+      tags:
+        - "Alert subscriptions"
+      summary: List alert subscription information for a particular query and alert type
+      description: This request searches all alerts for the query ID and alert type provided, and returns the associated alert subscription information.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/User-Agent'
+
+        - name: assetId
+          description: A query or schedule ID that you want the subscription information for.
+          required: true
+          in: path
+          type: string
+        - name: alertType
+          description: This property describes the state of query execution that triggers an alert. The response will only include alert subscription information for alerts of this type. The alert type is one of three values (start, success, failure).
+          required: true
+          in: path
+          type: string
+
+      responses:
+        200:
+          description: A successful response returns the user email IDs and notification channels for a particular query and alert type.
+          schema:
+            $ref: '#/definitions/subscribers_for_alert_by_assetId_and_alertType'
+        400:
+          description: "Bad Request. An invalid alertType was provided in the URL."
+          schema:
+            $ref: '#/definitions/get_alert_subscription_for_asset_id_and_alert_type_bad_request_error_message'
+        401:
+          description: "Unauthorized"
+        404:
+          description: "No alert was found for the given assetId."
+          schema:
+            $ref: '#/definitions/get_alert_subscription_for_asset_id_and_alert_type_not_found_error_message'
+        500:
+          description: "Internal Server Error."
+    patch:
+        operationId: "patchAlert"
+        tags:
+          - "Alert subscriptions"
+        summary: Enable or disable an alert
+        description: Update the status of an alert to either `enable` or `disable` by providing a query or schedule ID and an alert type.
```

Comment:
We're being a bit vague by saying "by providing a query...". Providing how? In the request path? A header? The payload?